internal/pkg/plugin/trello/update.go

```diff
@@ -0,0 +1,28 @@
+package trello
+
+import (
+	"github.com/merico-dev/stream/pkg/util/log"
+)
+
+// Update remove and set up trello workflows.
```

Comment:
Wrong comment? I don't think this plugin has anything to do with workflow any more.